src/runner.rs

```diff
@@ -22,9 +22,10 @@ pub async fn run_test_command(
     let start = SystemTime::now();
     let exit_code = run_test_and_get_exit_code(command, args).await?;
     log::info!("Command exit code: {}", exit_code);
-    let (file_sets, _file_counter) = get_files(repo, output_paths, team.clone(), codeowners_path.clone())?;
+    let (file_sets, _file_counter) =
+        get_files(repo, &output_paths, team.clone(), codeowners_path.clone())?;
     let failures = if exit_code != EXIT_SUCCESS {
-        get_failures(file_sets, start).await?
+        get_failures(&file_sets, Some(start)).await?
```

Comment:
`SystemTime::now()` should still be inline here to prevent the needless system call